internal/interceptors/auth_test.go

```diff
@@ -194,6 +186,6 @@ func Test_AppAuth_Failure5(t *testing.T) {
 	mockCore := mocks1.NewMockICore(ctrl)
 	mockCore.EXPECT().Get(gomock.Any(), project, username).Return(dummyCreds, nil)
 
-	ctx, err := AppAuth(newCtx, mockCore)
+	ctx, err := AppAuth(newCtx, mockCore, "")
```

Comment:
This test case is checking for `wrong password sent in authorizationHeaderKey` being sent. Let's keep the intent of the `TC` clear and send a non-empty valid `projectID` here.
Check other places as well in the file. Add failures on wrong/empty `projectID` as separate tests and make sure those tests specifically fail only the `projectID` checks.